dlls/wscript

```diff
@@ -27,6 +27,21 @@ def build(bld):
 	])
 
 	source += [
+		'gearbox/blkop_apache.cpp', 'gearbox/blkop_osprey.cpp', 'gearbox/cleansuit_scientist.cpp',
```

Comment:
Check if you can do bld.path.ant_glob(gearbox/*.cpp) instead.

I didn't used glob here because some files are just sample code.